src/core/connection.h

```diff
@@ -689,6 +689,30 @@ QuicConnGetNextExpirationTime(
     return Connection->Timers[0].ExpirationTime;
 }
 
+//
+// Returns the current QUIC version of the connection.
+//
+inline
+uint32_t
+QuicConnGetVersion(
```

Comment:
This helper seems a little unnecessary.

Reply:
As-is, I agree. but I was planning to make a follow-up PR where I replace the long-form with this function. The goal being to make it easier to tell where the version is being retrieved, versus where it is being set.

Reply:
As-is, I agree. but I was planning to make a follow-up PR where I replace the long-form with this function. The goal being to make it easier to tell where the version is being retrieved, versus where it is being set.

Reply:
All you are doing is checking if a variable == 2. While I agree it's a few more characters, I don't think it's worth the abstraction. @thhous-msft want to be the tie breaker here?

Reply:
I'll remove it and do it in another PR.